kivymd/tools/release/make_release.py

```diff
@@ -277,7 +285,10 @@ def main():
     # branches_to_push.append("stable")
 
     create_unreleased_changelog(
-        changelog_index_file, changelog_unreleased_file, version, test=test,
+        changelog_index_file,
+        changelog_unreleased_file,
+        version,
+        test=test,
     )
     update_init_py(next_version, is_release=False, test=test)
     git_commit(f"KivyMD {next_version}")
```

Comment:
Updated by pre-commit of github actions. 8994a39cd2b1050aa6b8342b533d148cbe2e065a